editoast/osm_to_railjson/README.md

```diff
@@ -4,14 +4,14 @@
 
 Example for Germany:
 
-1. Download OSM's Germany data (~4Gb `germany-latest.osm.pbf`) from
+1. Download OSM's German data (~4Gb `germany-latest.osm.pbf`) from
     https://download.geofabrik.de/europe/germany.html
-2. Launch osrd conversion (release build of editoast and conversion can be long):
+2. Launch conversion (release build of editoast and conversion can be long):
     ```sh
     cd ../../editoast
     cargo run -r -- osm-to-railjson <path/to/germany-latest.osm.pbf> <path/to/germany_railjson.json>
     ```
-3. Load railjson using the script (or through OSRD's web interface):
+3. Load railjson (also possible through a script or OSRD's web interface):
     ```sh
-    ../../scripts/load-railjson-infra.sh "germany" <path/to/germany_railjson.json>
+    cargo run -r -- infra import-railjson "Germany" <path/to/germany_railjson.json>
```

Comment:
Sorry, forgot it in my first comment 🤦 
```suggestion
    cargo run -r -- infra import-railjson  -g "Germany" <path/to/germany_railjson.json>
```

Reply:
Even better 😄 
```suggestion
    cargo run --release -- infra import-railjson --generate "Germany" <path/to/germany_railjson.json>
```

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/8825/commits/d7b9271eab6555228d00ef4c323cfb035ba06439